cmd/kubectl-gadget/utils/k8s.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/configflags"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/k8sutil"
+)
+
+func GetRunningGadgetNamespace() (string, bool) {
+	client, err := k8sutil.NewClientsetFromConfigFlags(configflags.KubernetesConfigFlags)
+	if err != nil {
+		log.Warnf("error creating k8s client: %s", err)
+		return "", false
+	}
+
+	nslist, err := client.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{
+		LabelSelector: "k8s-app=gadget",
+	})
+	if err != nil {
+		log.Warnf("error listing namespaces: %s", err)
+		return "", false
+	}
+
+	if len(nslist.Items) == 0 {
+		return "", false
+	} else if len(nslist.Items) > 1 {
+		log.Warnf("more than one gadget namespace found: %v", nslist.Items)
```

Comment:
The result of this message is difficult to understand:

```
WARN[0000] more than one gadget namespace found: [{{ } {bar    8f4258d6-bfa9-403b-aa77-06965776f871 6613332 0 2023-11-30 17:10:00 +0000 UTC <nil> <nil> map[k8s-app:gadget kubernetes.io/metadata.name:bar] map[] [] [] [{kubectl-gadget Apply v1 2023-11-30 17:10:00 +0000 UTC FieldsV1 {"f:metadata":{"f:labels":{"f:k8s-app":{}}},"f:spec":{}} }]} {[kubernetes]} {Active []}} {{ } {foo    38caf879-9e30-4a88-958e-10ab6a10178a 6612706 0 2023-07-10 21:22:27 +0000 UTC <nil> <nil> map[k8s-app:gadget kubernetes.io/metadata.name:foo] map[] [] [] [{kubectl-gadget Apply v1 2023-11-30 17:04:10 +0000 UTC FieldsV1 {"f:metadata":{"f:labels":{"f:k8s-app":{}}},"f:spec":{}} } {kubectl-create Update v1 2023-07-10 21:22:27 +0000 UTC FieldsV1 {"f:metadata":{"f:labels":{".":{},"f:kubernetes.io/metadata.name":{}}}} }]} {[kubernetes]} {Active []}}] 
K8S.NODE               K8S.NAMESPACE          K8S.POD                K8S.CONTAINER          PID        PPID       COMM        RET ARGS                        
Error: running gadget: getting target nodes: get gadget pods: no gadget pods found. Is Inspektor Gadget deployed?
```

I think it needs to print explicitely the namespaces names.